apps/src/applab/applab.js

```diff
@@ -586,6 +592,11 @@ Applab.init = function(config) {
         e.stop();
       });
     }
+    
+    if (studioApp.share) {
+      // automatically run in share mode:
+      window.setTimeout(studioApp.runButtonClick, 0);
```

Comment:
studioApp.runButtonClick.bind(studioApp), to not break references to "this" in turtle / Artist ?